allennlp_models/rc/models/dialog_qa.py

```diff
@@ -515,7 +518,9 @@ def _get_best_span_yesno_followup(
         if span_start_logits.dim() != 2 or span_end_logits.dim() != 2:
             raise ValueError("Input shapes must be (batch_size, passage_length)")
         batch_size, passage_length = span_start_logits.size()
-        max_span_log_prob = [-1e20] * batch_size
+
+        very_negative_constant = max(torch.finfo(span_start_logits.dtype).min, -1e20)
```

Comment:
This one can still just call [this function](https://github.com/allenai/allennlp/blob/32bccfbdaf97045f31861ab16bcfdefb8007c3f2/allennlp/nn/util.py#L1871), right?